plugins/lookup/hashi_vault.py

```diff
@@ -96,6 +96,7 @@
         - ldap
         - approle
         - aws_iam_login
+        - jwt
```

Comment:
Please add `C(jwt) is added in community.general 1.3.0.` to the description.

Reply:
Done